src/App.test.js

```diff
@@ -1,9 +1,36 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { shallow } from 'enzyme';
 import App from './App';
+import Header from './Header';
+import PokemonDetail from './PokemonDetail'
+// import SearchBar from './SearchBar'
+// import SearchResults from './SearchResults'
 
 test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  const wrapper = shallow(<App />);
+  expect(wrapper).toMatchSnapshot();
 });
+
+test('renders learn react link', () => {
+  const wrapper = shallow(<Header />);
```

Comment:
These should be in their own test files...Header.test.js and PokemonDetail.test.js